lib/iris/fileformats/nimrod.py

```diff
@@ -87,46 +89,95 @@
     "sat_space_count",
     "ducting_index",
     "elevation_angle",
+    "neighbourhood_radius",
```

Comment:
It would be nice to have a way ignore elements that, like setting values of a tuple to `_` as there are lots of names in these lists we don't use. Seek is used to ignore big blocks of unused elements though which is maybe good enough for now.